go.mod

```diff
@@ -134,7 +134,7 @@ require (
 	github.com/rancher/kubernetes-provider-detector v0.1.5
 	github.com/rancher/lasso v0.0.0-20240924233157-8f384efc8813
 	github.com/rancher/machine v0.15.0-rancher125
-	github.com/rancher/norman v0.0.0-20241001183610-78a520c160ab
+	github.com/rancher/norman v0.5.1
```

Comment:
Looking at https://github.com/rancher/norman/blob/main/VERSION.md, the norman version should be in v0.4.X, not v0.5.X

Reply:
Ope you're right. We'll need to go and backport in norman as well

Reply:
moved to `v0.4.1`